Source/SwiftLintBuiltInRules/Rules/Style/NonFinalClassRule.swift

```diff
@@ -0,0 +1,51 @@
+import SwiftSyntax
+import SwiftSyntaxBuilder
+
+@SwiftSyntaxRule(correctable: true, optIn: true)
+struct NonFinalClassRule: Rule {
+    var configuration = SeverityConfiguration<Self>(.warning)
+
+    static let description = RuleDescription(
```

Comment:
We now support adding rationals to rule descriptions. This feels like a very good candidate for some more explanation, as it seems highly opinionated and project-specific considering that `open` implies `public` which is often not what libraries want instead.